src/components/media-player-controls.js

```diff
@@ -82,6 +78,12 @@ function MediaPlayerControls({
     return <Menu items={items} onClick={handleSettingsMenuItemClick} />;
   };
 
+  const renderTimeDisplay = () => {
+    return durationInMilliseconds
+      ? <Fragment>{formatMillisecondsAsDuration(playedMilliseconds)}&nbsp;/&nbsp;{formatMillisecondsAsDuration(durationInMilliseconds)}</Fragment>
+      : <Fragment>--:--&nbsp;/&nbsp;--:--</Fragment>;
```

Comment:
Nice placeholder idea.

Reply:
Thanks, I thought we should indicate somehow that the file is not "loaded" yet (and I found "0:00" total length confusing) :-)